nih-sparc/sparc-app
# Add Download Option To All Datasets CU-cuj2gt

# Description

The purpose of this PR is to allow the download option in the `Get Dataset` dialog for all datasets. 

[Clickup Ticket](https://app.clickup.com/t/cuj2gt)
[Wrike Ticket](https://www.wrike.com/open.htm?id=529262647)

## Type of change


# How Has This Been Tested?

1. Navigate to a dataset that is less than 5GB in size.
2. Click on the `Get Dataset` button. A dialog should open with an option to download the dataset aside from the AWS option. Clicking on the download button should download the dataset.
3. Navigate to a dataset that is more than 5GB in size.
4. Click on the `Get Dataset` button. A dialog should open with messaging on how the dataset is too large and no download button. You should still see the AWS instructions.
5. Viewing the dialog in multiple viewports should maintain the size similar to how it appears on Discover.

# Checklist:
